### DISCUSSION TITLE
can I join the team undici

### DISCUSSION THREAD

**Discussion by mertcanaltin** [nodejs/undici#3087] [Q&A] [answered]
I'm curious about this, it's very fun to learn undiyi and make improvements

**Comment by Uzlopak** [comment, marked as answer]
I am not speaking for the others, but I think you need still some more PRs and code reviews.

**Reply by mertcanaltin** [reply]
I understand, thank you very much, I recently received an e-mail saying that I joined the team, I learned a lot from the team, I want to deserve to be in the team, if I have something missing, I can wait for the appropriate time @ronag 

thank you very much for your answers❤️

**Reply by Uzlopak** [reply]
Ok :)

I did not want to keep you unanswered. 

Welcome to the team ;)

**Reply by mertcanaltin** [reply]
Thank you very much 🙏
